src/experimental/ProbabilisticGraphicalModels/bayesian_network.jl

```diff
@@ -35,5 +36,54 @@ function BayesianNetwork{V}() where {V}
         Int[],
         BitVector(),
         BitVector(),
+        Symbol[],
     )
 end
+
+"""
```

Comment:
Function docstrings should be lead by the function and type signature, see https://docs.julialang.org/en/v1/manual/documentation/#Functions-and-Methods. 
```suggestion
"""
    add_stochastic_vertex!(bn::BayesianNetwork{V,T}, name::V, dist::Any,
                          is_observed::Bool=false, node_type::Symbol=:continuous)::T where {V,T}
```